Kitodo/src/main/java/org/kitodo/production/interfaces/activemq/MapMessageObjectReader.java

```diff
@@ -11,19 +11,23 @@
 
 package org.kitodo.production.interfaces.activemq;
 
+import java.util.Arrays;
 import java.util.Collection;
 import java.util.Collections;
 import java.util.HashMap;
 import java.util.HashSet;
+import java.util.List;
```

Comment:
```suggestion
```
`List` import unused when directly returning result of stream operation (see below)